### ISSUE TITLE
deepin system manager doesn't work?

### ISSUE THREAD

**Issue by prototype99** [zhtengw/deepin-overlay#40]
i installed it to try out and well... it doesn't do anything, or takes much too long to, anyway

**Comment by zhtengw** [comment]
Did you mean the package dde-extra/deepin-system-monitor cannot run?

**Comment by prototype99** [comment]
yep, i left it for 20 minutes and nothing happened, it just kept taking more and more ram

**Comment by zhtengw** [comment]
Could you please run it in terminal and paste the output?

**Comment by prototype99** [comment]
all i get is 
`"dtkwidget2" can not find qm files
"deepin-system-monitor" can not find qm files
Ethernet link detected`
would that be a dependency issue?

**Comment by zhtengw** [comment]
Nope, the INFOs are normal.

**Comment by prototype99** [comment]
any idea why?

**Comment by zhtengw** [comment]
Sorry, I have no idea now. Maybe you can remerge dde-base/dtkwidget to have a try.
I found that someone met the same issue (https://bbs.deepin.org/forum.php?mod=viewthread&tid=166037&extra=). But it seems that no one can help because the original developer of deepin-system-monitor dropt out of the Deepin Team.

**Comment by prototype99** [comment]
oh. that is indeed very unfortunate, i did give that a try (i remerged it specifically for this bug) however obviously to no success.

**Comment by zhtengw** [comment]
@prototype99 The deepin-system-monitor bump to new version 1.4.6, please upgrade and test.

**Comment by LuigiTheHunter** [comment]
The problem persists I have the newest version and the same bug.

**Comment by zhtengw** [comment]
I still cannot find any tips for this issue. 
Maybe you can start a bug report in [Deepin's github](https://github.com/linuxdeepin/developer-center/issues/new?template=bug_report.md) and discuss with the developer.
